src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+export const categorySlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => ({
+      categories: 'Under Construction',
+    }),
```

Comment:
- [ ] Please update the categories slice for the check status action it should return the string "Under construction"  and In case of unknown action - return the current state as requested in the project requirements.
  - You can consider the below code.
```javascript
 checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```

Reply:
Thank you @thecodechaser 😊